### PR TITLE
Add heroku documentation to README

### PR DESCRIPTION
**Pull request by evykassirer**
addresses #181

### CODE REVIEW

**Comment by evykassirer** [comment]
@ehashman and @fboxwala - would appreciate your comments on this cause idk if this the kind of information that you were thinking of or if it's explained clearly

thanks!

File: README.md [hunk 1]
```diff
@@ -270,6 +270,28 @@ To shut down the development server, use
 ./develop_server stop
 ```
 
+## Heroku Review Apps ##
```
Comment (annalorimer) [inline README.md:273]: Since this is used for testing I think it would make sense to put it in the testing section.

Comment (evykassirer) [inline README.md:273]: hmm what if I changed the above heading to `Testing Locally`?

Comment (ehashman) [inline README.md:273]: Yeah I agree that it doesn't make much sense to put this up above, as this is more CI-focused. The Testing Locally fix sounds fine to me.

File: README.md [hunk 2]
```diff
@@ -270,6 +270,28 @@ To shut down the development server, use
...
+
+Heroku is a deployment service we use to run test versions of our app. Heroku
+can make a unique URL for each open pull request, where you can view a
```
Comment (ehashman) [inline README.md]: Hmm, "unique URL" is a bit misleading. Let me try to reword this...

> When prompted, Heroku will provision temporary public web hosting and a domain name for any open pull request, and post a pingback on GitHub. This allows anyone to access the temporary site and review what our website would look like upon merging the changes from that pull request.

File: README.md [hunk 3]
```diff
@@ -270,6 +270,28 @@ To shut down the development server, use
...
+temporary version of the website with the changes from that pull request.
+
+### Permissions to access Heroku
+
+Access to Heroku isn't granted to everyone, but at least someone currently
+involved in the WiCS committee will ideally have the permissions. You don't
```
Comment (ehashman) [inline README.md]: This should probably reflect how we actually hand out access, which is

> Access to Heroku is granted to all committers in the wics-uw GitHub organization. Membership to the "committers" group is granted to all full Systems Committee members, as well as individuals that the Systems Committee have chosen in recognition of their contributions and demonstrated responsibility and good judgment.
> 
> To become a committer, you'll need to learn git, contribute ... , and then ask the Systems Committee for access, by emailing `<wics-sys@lists.uwaterloo.ca>`.

^ I added the backticks there to show how to add an email in Markdown.

File: README.md [hunk 4]
```diff
@@ -270,6 +270,28 @@ To shut down the development server, use
...
+git, contribute regularly to the WiCS website (both opening pull requests and
+reviewing them), and then ask someone on WiCS syscom for access (either on
+GitHub or by mailing ```wics-sys@lists.uwaterloo.ca```)
+
+### To deploy a pull request
+
+1. Go to [https://dashboard.heroku.com/apps](https://dashboard.heroku.com/apps)
+2. Click on ```wics-site```
+3. In the left column, named ```Review Apps```, find the PR you want to deploy
+   and click the button that says ```Create Review App```
```
Comment (ehashman) [inline README.md]: Triple backticks add in a newline and `<code>` formatting. I think here we just want asterisks, tbh.

Comment (evykassirer) [inline README.md]: I like the code formatting to show it as a clear label (I see it as a way of quoting something clearly, and the square around code formatted text feels similar to the way a button looks to me) but maybe I'm misusing it. Also weird, the markdown editor I was using showed triple backticks same as single, but I'll update that :)

File: README.md [hunk 5]
```diff
@@ -270,6 +270,32 @@ To shut down the development server, use
...
+
+Access to Heroku is granted to all committers in the wics-uw GitHub
+organization. Membership to the "committers" group is granted to all full
+Systems Committee members, as well as individuals that the Systems Committee
+have chosen in recognition of their contributions and demonstrated
+responsibility and good judgment.
+
+To become a committer, you'll need to learn git, contribute regularly to the
+WiCS website (both opening pull requests and reviewing them), and then ask the
+Systems Committee for access, by emailing `wics-sys@lists.uwaterloo.ca`.
```
Comment (ehashman) [inline README.md]: Again, you'll want angle brackets around this email, not backticks.

Comment (evykassirer) [inline README.md]: oh woops - fixed!

**Comment by ehashman** [comment]
Since the comment is now hidden:

> I like the code formatting to show it as a clear label (I see it as a way of quoting something clearly

Backticks and code formatting should be reserved for things that require that formatting for clarity. It should not be used for general emphasis---that's what asterisks/bold are for. Backticks should be reserved for cases where they convey additional meaning/clarity.

For example, command and code samples typically use triple backticks for legibility (triple backticks denote a large block that may contain newlines and should be rendered as a separate paragraph, and can use code hilighting, like this: ...)

``` clojure
{:hello (if Tuesday? "world" "moto")}
=> {:hello "moto"}
```

or single backticks for conveying additional meaning. We tend to use single backticks for directories, program names, user and Unix group names, etc. This helps to distinguish `git` (software for distributed version control) from git (a jerk) without providing additional written context.

**Comment by fboxwala** [comment]
lgtm

**Pull request merged**